docker/examples/docker-compose-eureka.yml

```diff
@@ -34,6 +34,33 @@ services:
       service: zipkin
     environment:
       - EUREKA_SERVICE_URL=http://eureka:8761/eureka/v2
+      - EUREKA_HOSTNAME=zipkin
     depends_on:
       eureka:
         condition: service_healthy
+
+  # Generate traffic by hitting http://localhost:8081
+  frontend:
+    image: ghcr.io/openzipkin/brave-example:armeria
```

Comment:
hard-coding to armeria as it is so easy to use Eureka (or consul, zookeeper, dns or their snazzy [central-dogma](https://line.github.io/centraldogma/))